docs/src/main/sphinx/release/release-426.md

```diff
@@ -0,0 +1,39 @@
+# Release 426 (30 Aug 2023)
+
+## General
+
+* Fix event listener incorrectly reporting output columns for `UPDATE`
+  statements with subqueries. ({issue}`18815`)
+
+## Security
+
+* Fix authentication failure with OAuth 2.0 when authentication tokens are
+  larger than 4 KB. ({issue}`18836`)
+
+## Delta Lake connector
+
+* Add support for the `TRUNCATE TABLE` statement. ({issue}`18786`)
+* Add support for `CASCADE` option in `DROP SCHEMA` statements. ({issue}`18333`)
+* Fix writing an incorrect transaction log for partitioned tables with an `id`
+  or `name` column mapping mode. ({issue}`18661`)
+
+## Hive connector
+
+* Add the `hive.metastore.thrift.batch-fetch.enabled` configuration property,
+  which can be set to `false` to disable batch metadata fetching from the Hive
+  metastore. ({issue}`18111`)
+* Fix `ANALYZE` failure when row count stats are missing. ({issue}`18798`)
+* Fix the `hive.target-max-file-size` configuration property being ignored
```

Comment:
Fix ... being ignored .. is kinda odd. Did we do that in the past as well..

Alternatively we could do 

Honor the `hive.target-max-file-size` configuration property when writing to sorted tables

but thats also odd .. Take the ... into account? 

Or 

Avoid ignoring the property...

not sure overall

Reply:
I think I like this best. I try to keep the first word to "Add/Support," "Improve," or "Fix" as often as possible so it's extremely clear what kind of change is happening. "Honor" is vague, "avoid ignoring" is a double negative, and I think I'd stand by this wording.

Reply:
We could also phrase it as

> Support the `xxx` configuration property when writing to sorted tables.

If it never worked, then this is a newly added feature. Although users expect it to work since it wasn't documented to not work, so it's also kind of a fix...

Reply:
I think what's weird about this phrasing is treating "xxx config property being ignored" as the target of what is being fixed. This is one of those cases where "Fixed an issue where the xxx was ignored" might be an improvement. But we always avoid the "fixed an issue" phrasing, so... I'm fine with the original.